qml/ThemeColors.qml

```diff
@@ -0,0 +1,42 @@
+/*
+ * <one line to give the program's name and a brief idea of what it does.>
```

Comment:
Ehm... please copy the header from some other file in this project.

Reply:
Not sure if the copyright by canonical 2013-2014 is really correct for the new file(?) but done.